### DISCUSSION TITLE
Finally have it up and running :)

### DISCUSSION THREAD

**Discussion by rooftopsolarsa** [ArminJo/JK-BMSToPylontechCAN#26] [Show and tell]
Thank You ArminJo

Looking good so far, everything working as expected.
Circled in Red is where I hooked up your project and connected to my inverter.

![jkbms-to-sunsynk-can-comms](https://github.com/ArminJo/JK-BMSToPylontechCAN/assets/153637794/6a267660-83ae-4b2f-88cc-da65db9a30f9)

**Comment by rooftopsolarsa** [comment]
![jkbms-comms-to-sunsynk5k](https://github.com/ArminJo/JK-BMSToPylontechCAN/assets/153637794/b97bab97-0299-489b-a556-f36c628443eb)

**Reply by ArminJo** [reply]
Thank you for sharing 👍

**Comment by rooftopsolarsa** [comment]
Running 3 days now without any issues. 

![running-3-days-now](https://github.com/ArminJo/JK-BMSToPylontechCAN/assets/153637794/6ec1b22a-6603-40b8-828a-db14dd3f8ddc)
